### PR TITLE
Add installing multiple plugins example

### PR DESCRIPTION
**Pull request by 10000-ki**
### Description

I think it would be helpful for users to have an example document that can install multiple plugins at once.
so i added

### Issues Resolved

None

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks for the contribution, @10000-ki! 
@peterzhuamazon - Can you please do a tech review?
@Naarcha-AWS -Can you please do a doc review?
Thanks, all.

File: _install-and-configure/plugins.md [hunk 1]
```diff
@@ -205,6 +205,20 @@ Continue with installation? [y/N]y
 Restart your OpenSearch node after installing a plugin.
 {: .note}
 
+### Installing multiple plugins
+
+Multiple plugins can be installed in one invocation
```
Comment (vagimeli) [inline _install-and-configure/plugins.md:210]:
```suggestion
Multiple plugins can be installed in one invocation.
```

File: _install-and-configure/plugins.md [hunk 2]
```diff
@@ -205,6 +205,20 @@ Continue with installation? [y/N]y
...
+
+#### Usage:
```
Comment (vagimeli) [inline _install-and-configure/plugins.md:212]:
```suggestion
#### Usage
```

File: _install-and-configure/plugins.md [hunk 3]
```diff
@@ -205,6 +205,20 @@ Continue with installation? [y/N]y
...
+```bash
+bin/opensearch-plugin install <plugin-name> <plugin-name> ... <plugin-name>
+```
+
+#### Example:
```
Comment (vagimeli) [inline _install-and-configure/plugins.md:217]:
```suggestion
#### Example
```

**Review by vagimeli — APPROVED** [review]
LGTM, with minor edits

**Comment by 10000-ki** [comment]
@vagimeli thank you for the comments
I applied all your feedback

File: _install-and-configure/plugins.md (hunk 2 above)
Comment (10000-ki) [reply _install-and-configure/plugins.md]: When I looked at the other docs, they were putting `:` on the back.
Is it okay to remove this?

Comment (10000-ki) [reply _install-and-configure/plugins.md]: https://github.com/opensearch-project/documentation-website/blob/9d3a738b218250ea912a401f7cdf0a6c8189729b/_install-and-configure/plugins.md?plain=1#L35

like this

**Review by peterzhuamazon — APPROVED** [review]

**Review by vagimeli — APPROVED** [review]

**Pull request merged**